bundles/specmate-emfrest/src/com/specmate/emfrest/authentication/Login.java

```diff
@@ -51,4 +40,15 @@ public void setAuthService(IAuthenticationService authService) {
 	public void setLogService(LogService logService) {
 		this.logService = logService;
 	}
+
+	@Override
+	protected UserSessionAndUser getUserSessionAndUser(Object object, Object object2, String token)

```

Comment:
please choose better names for object and object2 (does this come from the interface? maybe we should chagne the names there, too)

Reply:
It's coming from the interface and I honestly do not know what the arguments really are. I'll find out and change it!